scripts/download/lima.mjs

```diff
@@ -10,7 +10,7 @@ const limaRepo = 'https://github.com/rancher-sandbox/lima-and-qemu';
 const limaTag = 'v1.3';
 
 const alpineLimaRepo = 'https://github.com/lima-vm/alpine-lima';
-const alpineLimaTag = 'v0.1.2';
+const alpineLimaTag = 'v0.1.3';
 const alpineLimaEdition = 'std';
```

Comment:
The edition will need to be changed to `rd` (and I need to publish an ISO for that).

Reply:
@jandubois Do we need the edition change for this PR or for something in the future?

Reply:
Only the `rd` edition has nerdctl and buildkit pre-installed... I'll let you know once I've published the ISO.

Reply:
updated.